src/xrpl-account.js

```diff
@@ -183,6 +183,43 @@ class XrplAccount {
         return this.#submitAndVerifyTransaction(tx, options);
     }
 
+    /**
+     * 
+     * @param {*} signerList Ex:  [ {account:"ras24cvffvfbvfbbt5or4332", weight: 1}, {}, ...]
+     * @param {*} options  Ex:  {SignerQuorum: 1, sequence: 6543233}
+     * @returns a promise
+     */
+    setSignerList(signerList, options = {}) {
+        if (options.SignerQuorum < 1)
+            throw ("Everpocket: quorum must be greater than 0.");
+
+        let totalWeight = 0;
+        for (const signer of signerList) {
+            if (!(signerList.length > 0 && signer.account && signer.account.length > 0 && signer.weight && signer.weight > 0))
```

Comment:
Shouldn't this come before for loop, Because this check will never be true inside for loop